easy_phi/tests/run_tests.py

```diff
@@ -1,12 +1,46 @@
 # -*- coding: utf-8 -*-
-
+import tornado.httpserver
+import tornado.httpclient
+import tornado.ioloop
+import tornado.web
 import unittest
+import json
+from easy_phi import app as handlers
 
 class API_test(unittest.TestCase):
 
     def test_t(self):
         self.assertTrue(True)
 
+class TestTornadoWeb(unittest.TestCase):
+    http_server = None
+    response = None
+
+    def setUp(self):
+        application = tornado.web.Application([
+                (r'/api/v1/', handlers.PlatformInfoHandler),
```

Comment:
I need a way to start Tornado server, then run my tests and then stop Tornado server. This way it provides full cycle for test suite.
Here is where I found that approach:
https://groups.google.com/forum/#!msg/python-tornado/hnz7JmXqEKk/MNVwkarn7m4J